### DISCUSSION TITLE
Contributors general discussion

### DISCUSSION THREAD

**Discussion by pczarn** [contain-rs/bit-vec#113] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with `bit-vec`.

**Comment by pczarn** [comment]
@Jujumba We may comment about absolutely anything in this space

**Comment by pczarn** [comment]
@Jujumba how did you start programming? Would you like to share what you've been doing in Rust and otherwise?

I have started a "curated list of libraries" for algorithms and structures like bit-vec but not limited to containers like `contain-rs`. It's called `omnitool` and unlike here, it should go all in one repository. It would be "Awesome Algorithms and Data Structures for Rust" but all as forks of repos plus maintenance and larger ambitions. 

https://github.com/pax-soft/omnitool

**Reply by Jujumba** [reply]
> @Jujumba how did you start programming?

Out of boredom 2 years ago :)

> Would you like to share what you've been doing in Rust and otherwise?

What do you mean? My Rust code is available at Github 😅 

> I have started a "curated list of libraries" for algorithms and structures like bit-vec but not limited to containers like contain-rs. It's called omnitool and unlike here, it should go all in one repository. It would be "Awesome Algorithms and Data Structures for Rust" but all as forks of repos plus maintenance and larger ambitions.
> 
> https://github.com/pax-soft/omnitool

Sound interesting and promising 👀
